src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,18 @@
 package core.basesyntax;
 
 public class PasswordValidator {
-    public void validate(String password, String repeatPassword) {
-        //write your code here
+    public void validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        try {
+            if (password.length() < 10 || repeatPassword.length() < 10) {
+                throw new PasswordValidationException("Wrong passwords");
+            } else if (password.length() == 0 || repeatPassword.length() == 0) {
+                throw new PasswordValidationException("Wrong passwords");
+            } else if (!password.equals(repeatPassword)) {
+                throw new PasswordValidationException("Wrong passwords");
+            }
+        } catch (NullPointerException e) {
+            throw new PasswordValidationException("Your passwords are incorrect. Try again.");
+        }
```

Comment:
1. Instead of try/catch block it's better to add one validation for null in the beginning
if (password == null || repeatPassword == null) {...

2. else if (password.length() == 0 || repeatPassword.length() == 0)
is it possible after check (password.length() < 10 || repeatPassword.length() < 10) ?